### PR TITLE
Replaced dropped text, fixed some formatting

### PR DESCRIPTION
**Pull request by rpetrusha**

### CODE REVIEW

File: xml/System.Windows.Controls/StickyNoteControl.xml [hunk 1]
```diff
@@ -665,9 +665,10 @@
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- The  identifier for the <xref:System.Windows.Controls.StickyNoteControl.IsExpanded%2A> dependency property.
```
Comment (carlossanlop) [inline xml/System.Windows.Controls/StickyNoteControl.xml]: Was there a dropped word/link here?

Comment (rpetrusha) [reply xml/System.Windows.Controls/StickyNoteControl.xml]: No. I thought perhaps there was, but this is a standard remark for other StickyNoteControl dependency properties.

**Comment by rpetrusha** [comment]
They were dropped when we migrated from DxStudio to CAPS, which was our authoring system before we migrated to GitHub, @carlossanlop. There was a bug in the migration routine.

File: xml/System.Workflow.Runtime/WorkflowRuntime.xml [hunk 2]
```diff
@@ -1653,21 +1653,19 @@ static void Main(string[] args)
...
   
- For the <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event, the sender contains the <xref:System.Workflow.Runtime.WorkflowRuntime> and <xref:System.Workflow.Runtime.WorkflowEventArgs> contains the <xref:System.Workflow.Runtime.WorkflowInstance> associated with the event.  
+The workflow run-time engine raises the `WorkflowUnloaded` event after the state of the workflow instance has been successfully persisted but before the instance is invalidated in memory. Therefore, a <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowPersisted> event precedes the <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event.  
   
- For more information about handling events, see [Consuming Events](https://go.microsoft.com/fwlink/?LinkID=157352).  
+For the `WorkflowUnloaded` event, the sender contains the <xref:System.Workflow.Runtime.WorkflowRuntime> and <xref:System.Workflow.Runtime.WorkflowEventArgs> contains the <xref:System.Workflow.Runtime.WorkflowInstance> associated with the event.  
   
-   
+For more information about handling events, see [Consuming Events](https://go.microsoft.com/fwlink/?LinkID=157352).
```
Comment (mairaw) [inline xml/System.Workflow.Runtime/WorkflowRuntime.xml]: not related to your PR, but this link goes to a very old topic. We do have an issue to review FwLinks. This just reinforces that.

File: xml/System/InvalidOperationException.xml [hunk 3]
```diff
@@ -89,7 +89,7 @@
...
   
--   In UWP apps, call the [CoreDispatcher.HasThreadAccess](https://msdn.microsoft.com/library/windows/apps/windows.ui.core.coredispatcher.hasthreadaccess.aspx) method to determine if a method is running on a non-UI thread. Call the  [CoreDispatcher.RunAsync](https://msdn.microsoft.com/library/windows/apps/windows.ui.core.coredispatcher.runasync.aspx) method to execute a delegate that updates the UI thread. Use the  
+-   In UWP apps, call the [CoreDispatcher.HasThreadAccess](https://msdn.microsoft.com/library/windows/apps/windows.ui.core.coredispatcher.hasthreadaccess.aspx) method to determine if a method is running on a non-UI thread. Call the [CoreDispatcher.RunAsync](https://msdn.microsoft.com/library/windows/apps/windows.ui.core.coredispatcher.runasync.aspx) method to execute a delegate that updates the UI thread.
```
Comment (mairaw) [inline xml/System/InvalidOperationException.xml]: Unrelated to your changes as well:

```suggestion
-   In UWP apps, check the <xref:Windows.UI.Core.CoreDispatcher.HasThreadAccess?displayProperty=nameWithType> property to determine if a method is running on a non-UI thread. Call the <xref:Windows.UI.Core.CoreDispatcher.RunAsync%2A?displayProperty=nameWithType> method to execute a delegate that updates the UI thread. 
```

**Comment by mairaw** [comment]
Yep, and then we had to start this migration before we even fixed all the issues from the previous migration, but we're getting there.

File: xml/System.Workflow.Runtime/WorkflowRuntime.xml (hunk 2 above)
Comment (rpetrusha) [reply xml/System.Workflow.Runtime/WorkflowRuntime.xml]: A good replacement is [Handling and raising events](~/docs/standard/events/index.md). I'll search for occurrences of **Consuming Events** and open a PR to replace them. I've replaced all 13 occurrences in this file.

**Pull request merged**